templates/azure/index.html

```diff
@@ -249,29 +172,134 @@ <h2>Get Ubuntu Pro on Azure now</h2>
           </div>
           <div class="col-3">
             <p>
-              <a href="/azure/pro">Learn more about Ubuntu Pro&nbsp;&rsaquo;</a>
+              <a href="/azure/pro">Read more about Ubuntu Pro&nbsp;&rsaquo;</a>
             </p>
           </div>
         </div>
       </div>
     </div>
   </section>
 
-  <section class="p-section--deep">
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <div class="p-section--shallow">
+        <hr class="p-rule--highlight" />
+        <h2 class="p-muted-heading">Hear from our partners</h2>
+      </div>
+      <div class="row--25-75 p-section--shallow">
+        <div class="col">
```

Comment:
You could use the [Vanilla quote macro](https://vanillaframework.io/docs/patterns/quote-wrapper#without-heading) for these (I think, I haven't tried it myself)